apps/extension/tests/fixtures/fixtures.ts

```diff
@@ -47,6 +47,16 @@ export const test = base.extend<TestFixtures>({
   page: async ({ context }, use) => {
     const pages = context.pages();
     const page = pages.length > 0 ? pages[0] : await context.newPage();
+    await page.addStyleTag({
+      content: `
+        *, *::before, *::after {
```

Comment:
Is waiting for the animations causing problems? It would be good to rebase the commits so each one explains the fixes applied as there are a few different things happening:
- mocks added
- animations disabled
- increased delays with `test.slow`